contracts/Vault.vy

```diff
@@ -1051,15 +1054,16 @@ def addStrategy(
     @param strategy The address of the Strategy to add.
     @param debtRatio The ratio of the total assets in the `vault that the `strategy` can manage.
     @param rateLimit
-        How many assets per block this Vault may deposit to or withdraw from
-        `strategy`.
+        Limit on the increase of debt per unit time since last harvest
     @param performanceFee
         The fee the strategist will receive based on this Vault's performance.
     """
     assert strategy != ZERO_ADDRESS
+    assert not self.emergencyShutdown
 
     assert msg.sender == self.governance
     assert self.debtRatio + debtRatio <= MAX_BPS
+    assert performanceFee < MAX_BPS 
```

Comment:
```suggestion
    assert performanceFee <= MAX_BPS - self.performanceFee
```

Reply:
don't understand the additional subtraction `- self.performanceFee`? can't the number be updated to any arbitrary value as long as its equal or below MAX_BPS